src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -46,6 +46,17 @@ public UniquePersonList(Person... persons) throws DuplicatePersonException {
         internalList.addAll(initialTags);
     }
 
+
+    @Override
+    public String getPrintableString (Printable... printables) {
+        String result = "";
```

Comment:
Using String is okay but you may want to consider using StringBuilder. String is immutable in Java. When we do String manipulation such as string concatenation, Java creates a new String and discard the older String for garbage collection. Multiple concatenations will generate a lot of garbage in the heap. We avoid this by using StringBuilder as StringBuilder is a mutable class.